var/spack/repos/builtin/packages/rocgdb/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rocgdb(AutotoolsPackage):
+    """This is ROCgdb, the ROCm source-level debugger for Linux,
+        based on GDB, the GNU source-level debugger."""
+
+    homepage = "https://github.com/ROCm-Developer-Tools/ROCgdb/"
+    url      = "https://github.com/ROCm-Developer-Tools/ROCgdb/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='cf36d956e84c7a5711b71f281a44b0a9708e13e941d8fca0247d01567e7ee7d1')
+    depends_on('cmake@3.5.2', type='build')
+    depends_on('texinfo')
+    depends_on('bison')
+    depends_on('flex')
+    depends_on('libunwind')
+    depends_on('expat')
+    depends_on('python')
```

Comment:
Python is often a build/run dependency. Do you actually link to the python libraries? Is it only needed at build-time?

Reply:
it was required for the build time. Will update accordingly.